### DISCUSSION TITLE
browser login

### DISCUSSION THREAD

**Discussion by 0drat** [mastodon/mastodon#20066] [Q&A]
I am signed in via the app but if click on Mastodon links / profiles in twitter it launches in the browser and wont let me sign in with my mastodon account, it says my email address is invalid when it is not.  I can access my Mastodon profile if i go to Mastodon.scot ?

**Comment by jem-needs-help** [comment]
I am having a similar issue.  I can't log in, it keeps telling me my email or password are invalid, but if I try to do the password recovery, nothing ever shows up in my email.  I'm basically locked out.

**Comment by beatriztoledodev** [comment]
same here :(

**Comment by afontenot** [comment]
I believe you are all just seeing the result of federation. On Mastodon (and the Fediverse more generally), each person has both a user name *and* a server. Just like you might have `yourusername` at Gmail and someone else might have that same username somewhere else. When you are clicking links to other people's Mastodon profiles, those links are taking you to *their* instances. When you try to sign in, it's like trying to sign in to Hotmail with your Gmail username and password. It won't work.

What you have to do to follow these people is to go to your home instance (whatever that may be) and then search for the `@username@instance` of the person you want to follow. **Or** if you are looking at someone else's profile on *their* instance, instead of trying to log in you should just click "follow". That will take you to a page where you can enter your own username and instance. Clicking "proceed to follow" will take you back to your home instance, where you will be able to follow the user.

**Comment by sarahbreau** [comment]
Ugh, me too, it's been 2 weeks and I can still only use the app. I really want to log in online so I can do things like pin my #introduction toot, and I would love to try some of the other apps like Tusky.
